proprietary/tokens/src/brand/amsterdam/space.compact.tokens.json

```diff
@@ -3,7 +3,10 @@
     "space": {
       "xs": { "value": "clamp(0.25rem, calc(0.390625vw - 0.015625rem), 0.625rem)" },
       "sm": { "value": "clamp(0.5rem, calc(0.78125vw - 0.03125rem), 1.25rem)" },
-      "md": { "value": "clamp(1rem, calc(1.5625vw - 0.0625rem), 2.5rem)" },
+      "md": {
+        "value": "clamp(1rem, calc(1.5625vw - 0.0625rem), 2.5rem)",
+        "comment": "Grows from 16px at 1088px wide to 40px at 2624px wide."
```

Comment:
Why does only 'md' have a description?

Reply:
Because `md` is the base value from which the other ones are derived. The comment has been copied over from the grid tokens, where ‘medium’ was the only one.

We could document the multiplication factors of ¼, ½, 1½, and 2, but I guess the minimum values already do so?